drivers/rtc/Kconfig.siwx91x

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2025 Silicon Laboratories Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+config RTC_SILABS_SIWX91X
+	bool "Silabs siwx91x RTC driver"
+	default y
+	depends on DT_HAS_SILABS_SIWX91X_RTC_ENABLED
+	help
+	  Build RTC driver for Silabs SIWX91X soc
```

Comment:
The Reference Manual talk about "Calendar hardware block" while Zephyr talk about "RTC". In addition, there is also another clock called "RTC" on SiWx91x. Maybe you can mention that in the help section (or in the dt-bindings).